src/status_im/contexts/wallet/sheets/network_preferences/view.cljs

```diff
@@ -105,4 +150,4 @@
                               :on-press            (fn []
                                                      (let [chain-ids (map :chain-id current-networks)]
                                                        (on-save chain-ids)))
-                              :customization-color color}}]]))))
+                              :customization-color (or color :blue)}}]]))))
```

Comment:
we really should just update the customization-color subs to have a default value? 🤔 

This goes beyond this pr ofc, we have this all over the codebase 😅

Reply:
I think this is not needed actually, just removed it